scripts/check_kernel_features.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
```

Comment:
Not sure if this is the correct place for the script

Reply:
Yep, this is good!